### PR TITLE
Added Instacart account removal information

### PR DESCRIPTION
**Pull request by MaKiPL**

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
Does the form provide immediate deletion or does it require user intervention?

File: _data/sites.json [hunk 1]
```diff
@@ -7930,6 +7930,15 @@
...
     {
+        "name": "Instacart",
+        "url": "https://www.instacart.com/user_privacy/form/",
+        "difficulty": "easy",
+        "domains": [
+            "instacart.com"
+        ]
+    },
+
+     {
```
Comment (tupaschoal) [inline _data/sites.json:7941]:
```suggestion
    {
```

**Comment by MaKiPL** [comment]
> Does the form provide immediate deletion or does it require user intervention?

Immediate- basically you login and click the button for account removal

**Review by tupaschoal — APPROVED** [review]
Great, thanks for the quick confirmation and the contribution!

**Pull request merged**